biome.json

```diff
@@ -1,33 +1,27 @@
 {
-  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "$schema": "https://biomejs.dev/schemas/1.9.2/schema.json",
   "organizeImports": {
     "enabled": true
   },
-  "formatter": {
-    "include": ["**/*.ts", "**/*.tsx", "**/*.json"],
+  "files": {
```

Comment:
Thank you friend, you are a friend!
![image](https://github.com/user-attachments/assets/4ac68a2d-a156-4bf6-bd90-957cff64996f)